Description: Fix some imports
 Some imports statements are wrong/not adapted for the way we build things.
Author: Julien BLACHE <jblache@debian.org>

Index: airport-utils-2/airportconfig/airport/AirportEncryptionPanel.java
===================================================================
--- airport-utils-2.orig/airportconfig/airport/AirportEncryptionPanel.java	2010-03-11 20:55:56.570753346 +0100
+++ airport-utils-2/airportconfig/airport/AirportEncryptionPanel.java	2010-03-11 20:56:29.457753544 +0100
@@ -25,9 +25,7 @@
 
 import java.awt.*;
 import javax.swing.*;
-import AirportBaseStationConfigurator;
-
-
+import airportconfig.*;
 
 
 /**
Index: airport-utils-2/airportconfig/airport/AirportInfo.java
===================================================================
--- airport-utils-2.orig/airportconfig/airport/AirportInfo.java	2010-03-11 20:55:56.582753518 +0100
+++ airport-utils-2/airportconfig/airport/AirportInfo.java	2010-03-11 20:56:29.469056742 +0100
@@ -25,8 +25,7 @@
 
 import java.util.*;
 import byteblock.*;
-import AirportBaseStationConfigurator;
-
+import airportconfig.*;
 
 /**
 * 	Collects together information about the location of pieces of information 
